shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -1234,6 +1234,26 @@ - (BOOL)accessibilityElementsHidden {
 
 @end
 
+@interface FlutterTimingProxy : NSObject
+@property(nonatomic, assign) id target;
+@end
+
+@implementation FlutterTimingProxy
+
++ (instancetype)proxyWithTarget:(id)target {
+  FlutterTimingProxy* proxy = [[self new] autorelease];
+  if (proxy) {
+    proxy.target = target;
+  }
+  return proxy;
+}
+
+- (void)enableActiveViewAccessibility {
```

Comment:
Can you add an NSAssert that the target is not dealloc'd?

Reply:
I don't think they are going to be able to do that without ARC.

Reply:
Yeah, seems we can do that without ARC.